.github/workflows/docs.yml

```diff
@@ -90,9 +90,9 @@ jobs:
           if [[ "$REF_KIND" == "tags" ]]; then
               TAG=${GITHUB_REF#refs/tags/}
               mv ../docs/build/html $TAG
-          else
-              rm -rf latest
-              mv ../docs/build/html latest
+          else # deploys directly to the root of the website
```

Comment:
I would remove the code handling git tags here, since the "normal" path would remove the tag on the next build anyway.

I can't do a suggestion, but this would just be replacing the `if` block with only

```
rm -rf *
mv ../docs/build/html/* .
```

Reply:
done, and also moved the post-build stuff to conf.py.
will merge now thanks for the feedback